client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -96,6 +96,44 @@ Undocumented.prototype.jetpackModulesDeactivate = function( siteId, moduleSlug,
 	return this.wpcom.req.post( { path: '/sites/' + siteId + '/jetpack/modules/' + moduleSlug }, {}, { active: false }, fn );
 };
 
+/*
+ * Retrieve the settings of a Jetpack module with moduleSlug on the site with id siteId
+ *
+ * @param {int} [siteId]
+ * @param {string} [moduleSlug]
+ * @param {Function} fn
+ * @api public
+ */
+Undocumented.prototype.fetchJetpackModuleSettings = function( siteId, moduleSlug, fn ) {
+	//@TODO: implement and test this endpoint, it's currently not working
+	return this.wpcom.req.get( {
+		path: '/jetpack-blogs/' + siteId + '/rest-api/',
+		body: {
+			path: '/module/' + moduleSlug
```

Comment:
Sure. In the declaration below, `path` is part of `query`
`query: {  path: '/module/' + moduleSlug }`

Perhaps, for consistency, `path` should be part of `query` here as well, instead of `body`